guardian/tagmanager
# Upgrade packages to address security vulnerabilities

## What does this change?

This repository has open PRs from Snyk for a number of security vulnerabilities. This PR upgrades (hopefully) all dependencies that are vulnerable (`npm audit fix`) or otherwise could use an upgrade (`npm upgrade`), which will allow us to close the open PRs if they are not automatically closed themselves.

## How to test

Clone the repo, spin up the app and see that everything works as expected!

Or, go to CODE where it is currently deployed.

## How can we measure success?

The app works exactly as before!
